ParkingZoneApp/Views/Reservation/Reserve.cshtml

```diff
@@ -0,0 +1,53 @@
+﻿@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+@model ParkingZoneApp.ViewModels.ReservationVMs.ReserveVM
+@{
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<h3>@Model.ZoneName, in @Model.ZoneAddress</h3>
+<hr />
+<h4>Reserve Parking Slot Number @Model.SlotNumber</h4>
+<div class="row">
+    <div class="col-md-4">
+        <form asp-action="Reserve">
+            <input type="hidden" value="@Model.SlotId" asp-for="SlotId" />
+            <input type="hidden" value="@Model.ZoneId" asp-for="ZoneId" />
+            <input type="hidden" value="@Model.ZoneName" name="ZoneName" />
+            <input type="hidden" value="@Model.ZoneAddress" name="ZoneAddress" />
+            <input type="hidden" value="@Model.SlotNumber" name="SlotNumber" />
```

Comment:
you dont need to pass these things because in the post action you can take them with id

Reply:
Oh yeah, previusly I was using different method and forgot to remove after chaging it